sys-apps/coreutils/files/coreutils-9.0-r1-0001-dircolors-consider-COLORTERM-sufficient-for-color.patch

```diff
@@ -0,0 +1,45 @@
+From f67ef065d7bad549fdb95d27d67dd21099b9541e Mon Sep 17 00:00:00 2001
```

Comment:
Could you put a link to upstream submission at the top please? But also, are they going to accept this?